src/wallet/scriptpubkeyman.h

```diff
@@ -254,9 +254,9 @@ class ScriptPubKeyMan
 
     /** Prepends the wallet name in logging output to ease debugging in multi-wallet use cases */
     template <typename... Params>
-    void WalletLogPrintf(const char* fmt, Params... parameters) const
+    void WalletLogPrintf(util::ConstevalFormatString<sizeof...(Params)> wallet_fmt, const Params&... params) const
     {
-        LogPrintf(("%s " + std::string{fmt}).c_str(), m_storage.GetDisplayName(), parameters...);
+        LogInfo("%s %s", m_storage.GetDisplayName(), tfm::format(wallet_fmt, params...));
```

Comment:
Should this always be info?
Some `WalletLogPrintf` usages seem like [warnings](https://github.com/bitcoin/bitcoin/blob/master/src/wallet/receive.cpp#L233-L234)), or [errors](https://github.com/bitcoin/bitcoin/blob/master/src/wallet/rpc/backup.cpp#L600).

Reply:
That's my concern as well, it seems to me we're changing behavior here by swapping a neutral `LogPrintf` to `Info` level logging, when in fact it's sometimes a `warning` or `error`.

Reply:
Got it, thanks.
Would it make sense to add an optional log-level parameter here to be able to set some of those logs to errors/warns in a future PR?